packages/trader/src/AppV2/Components/SymbolNotFound/symbol-not-found.tsx

```diff
@@ -0,0 +1,25 @@
+import { Text } from '@deriv-com/quill-ui';
+import { StandaloneSearchRegularIcon } from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+import React from 'react';
+
+const SymbolNotFound = ({ searchTerm }: { searchTerm?: string }) => {
+    return (
+        <div className='symbol-not-found__container'>
+            <StandaloneSearchRegularIcon
+                fill='var(--semantic-color-slate-solid-surface-static-midLowest)'
+                iconSize='2xl'
+            />
+            <div className='symbol-not-found--content'>
```

Comment:
nit-pick :):
```suggestion
            <div className='symbol-not-found__content'>
```
same as for __container above